ui/index.ts

```diff
@@ -91,7 +93,27 @@ import useNavigation from "./hooks/navigation";
 import { useListAlerts, useListProviders } from "./hooks/notifications";
 import { useGetObject, useListObjects } from "./hooks/objects";
 import { useListSources } from "./hooks/sources";
-import { Core as coreClient } from "./lib/api/core/core.pb";
+import {
+  GetObjectRequest,
+  GetObjectResponse,
+  IsCRDAvailableRequest,
+  IsCRDAvailableResponse,
+  ListObjectsRequest,
+  ListObjectsResponse,
+  GetPolicyRequest,
+  GetPolicyResponse,
+  GetPolicyValidationRequest,
+  GetPolicyValidationResponse,
+  ListPoliciesRequest,
+  ListPoliciesResponse,
+  GetSessionLogsResponse,
+  ListError,
+  LogEntry,
+  PolicyValidation,
+  PolicyObj,
+  PolicyParam,
+  Core as coreClient,
+} from "./lib/api/core/core.pb";
```

Comment:
All types are exported in the weave-gitops npm package underneath `/ui/` (would be neat to remove this prefix somehow / someday),  e.g. you can import these from a consuming app with:

```ts
import { GetObjectRequest } from '@weaveworks/weave-gitops/ui/lib/api/core/core.pb';
```

Note this is only for types, not for classes/object/functions etc.

Reply:
This is for the eslint rule import/no-unresolved 
https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md

At the bottom of that doc under "When to not use this", it says: "If you're using a module bundler other than Node or Webpack, you may end up with a lot of false positive reports of missing dependencies." Maybe that's what we have here?